File: Q7.java
import java.util.Scanner;
import java.text.DecimalFormat;

public class Q7
{
    public static Scanner leia = new Scanner(System.in);
    public static void main(String args[])
    {
        float número[] = new float[10], soma = 0;
        int vezes = 1;
        String somaf;
        for (int i = 0; i < número.length; i ++, vezes ++)
        {
            System.out.printf("Digite o %d° número: ", vezes);
            número[i] = leia.nextFloat();
            soma = soma + número[i];
        }
        somaf = new DecimalFormat("#.#").format(soma);
        System.out.printf("A soma dos 10 números que você digitou é %s.\n", somaf);
    }
}

File: Q7.cs
using System;

namespace Q7
{
    class Program
    {
        static void Main(string[] args)
        {
            float[] número = new float[10]; 
            float soma = 0;
            byte vezes = 1;
            for (byte i = 0; i < número.Length; i ++, vezes ++)
            {
                Console.Write("Digite o {0}° número: ", vezes);
                número[i] = float.Parse(Console.ReadLine());
                soma = soma + número[i];
            }
            Console.WriteLine("A soma dos 10 números que você digitou é {0}.", soma);
        }
    }
}